Source/WebKit/WebProcess/WebPage/WebFoundTextRangeController.cpp

```diff
@@ -513,7 +518,7 @@ WebCore::Document* WebFoundTextRangeController::documentForFoundTextRange(const
 
 std::optional<WebCore::SimpleRange> WebFoundTextRangeController::simpleRangeFromFoundTextRange(WebFoundTextRange range)
 {
-    return m_cachedFoundRanges.ensure(range, [&] () -> std::optional<WebCore::SimpleRange> {
+    auto foundRange = m_cachedFoundRanges.ensure(range, [&] () -> std::optional<WebCore::WeakSimpleRange> {
```

Comment:
Optional since this is pre-existing code but you can fix the style guide violation? i.e. no space between capture list and arg list.

In fact, the arg list can be omitted entirely now.

```suggestion
    auto foundRange = m_cachedFoundRanges.ensure(range, [&] -> std::optional<WebCore::WeakSimpleRange> {
```